packages/breadcrumb/README.md

```diff
@@ -0,0 +1,123 @@
+# @undercoatcsss/breadcrumb
+
+Backdrop is a component of the [Undercoat.css](https://github.com/undercoat/undercoat) framework.
+
+![Travis (.org)](https://img.shields.io/travis/undercoat/undercoat.svg?style=flat-square) ![npm (scoped)](https://img.shields.io/npm/v/@undercoatcss/breadcrumb?style=flat-square) ![WCAG 2.0 Level AA](https://img.shields.io/badge/WCAG_2.0-Level_AA-brightgreen.svg?style=flat-square) ![schema.org](https://img.shields.io/badge/microdata-schema.org-brightgreen.svg?style=flat-square)
+
+> **CAUTION! IT'S UNDER HEAVY DEVELOPMENT CURRENTLY! DON'T USE IT IN PRODUCTION!**
+
+## Description
+
+Breadcrumb is a logically and functionally independent reusable **page component** (block), which represents a **secondary navigation scheme** that reveals the user's location in a website.
+
+## Installation
+
+### NPM
+
+```bash
+npm install @undercoatcss/breadcrumb
+```
+
+## Usage
+
+After installation, you can link or import the assets into your project:
+
+### CDN
+
+```html
+<link rel="stylesheet" href="https://unpkg.com/@undercoatcss/breadcrumb@latest/dist/breadcrumb.min.css">
+```
+
+### Webpack
+
+```javascript
+import '@undercoatcss/breadcrumb/breadcrumb.css';
+```
+
+## Settings
+
+The component provides CSS custom property to change its defaults.
+
+| Property                               | Default value
+| -                                      | -
+| --breadcrumb--separator                | \2192
+
+## HTML markup
+
+### Without microdata
+
+```html
+<nav class="breadcrumb" aria-label="Breadcrumb">
+  <ul class="breadcrumb__list" role="list">
```

Comment:
I think we don't need the `role="list"` for the native list elements, same for `role="list-item"`

There is a fix for the `list-style: none` and Safari, also, an explanation against ARIA roles:
[Fix + explanation](https://www.scottohara.me/blog/2019/01/12/lists-and-safari.html)
One more article with 2019 comments: 
[Fix](https://unfetteredthoughts.net/2017/09/26/voiceover-and-list-style-type-none/)

Reply:
Wow! Thx! Fixed!